sway/config/output.c

```diff
@@ -300,6 +317,9 @@ static struct output_config *get_output_config(char *name, char *identifier) {
 		snprintf(temp, length, "%s on %s", identifier, name);
 
 		result = new_output_config(temp);
+		if (config->reloading) {
+			default_output_config(result, sway_output->wlr_output);
+		}
```

Comment:
Can we move these lines to line 309, since we always to the same thing for all `if` branches?

Reply:
I refactored it to move those lines above the branches